src/chunk_dispatch.h

```diff
@@ -28,6 +28,10 @@ typedef struct ChunkDispatch
 	ResultRelInfo *hypertable_result_rel_info;
 	OnConflictAction on_conflict;
 	List	   *arbiter_indexes;
+	int			returning_index;
```

Comment:
Does it make sense to consider just passing a pointer to the plan or something, instead of recreating a lot of fields?

Reply:
I considered that but decided against that because I want to reduce the exposure of plan stuff inside execution nodes. I think this indirection helps decouple the two.